### PR TITLE
[MWPW-158469] P0 Changes For Feature Compare Table

### PR DESCRIPTION
**Pull request by echen-adobe**
**Fixes following issues|Adds following features:**

Optional greying of text + icons via italicizing the text 
CSS reformatting to support some column headers having a CTA and some columns not having a CTA
CSS variant vertical-header that rearranges the header so that items are displayed in a vertical fashion. 
First section of the page is already open on mobile, right now all sections are by default closed.

**Resolves:** https://jira.corp.adobe.com/projects/MWPW/issues/MWPW-158469

**Steps to test the before vs. after and expectations:**
- Visit the linked page and verify that functionality is the same for the new branch and the stage branch.

**Pages to check for regression and performance:**
-  https://pricing-compare-upgrades--express--adobecom.hlx.page/drafts/echen/individuals-and-teams-upgrade
-  https://pricing-compare-upgrades--express--adobecom.hlx.page/express/pricing

Screenshot 2024-09-18 at 10 50 10 AM
Screenshot 2024-09-18 at 10 50 14 AM

### CODE REVIEW

File: express/blocks/pricing-table/pricing-table.js [hunk 1]
```diff
@@ -67,6 +68,7 @@ function handleHeading(headingRow, headingCols) {
     const colLabel = document.createElement('div');
     colLabel.classList.add('col-heading');
     [...elements].forEach((e) => {
+      console.log(e);
```
Comment (fullcolorcoder) [inline express/blocks/pricing-table/pricing-table.js:71]: Remove

**Review by fullcolorcoder — CHANGES_REQUESTED** [review]
1 small thing, looks good otherwise

File: express/blocks/pricing-table/pricing-table.js (hunk 1 above)
Comment (echen-adobe) [reply express/blocks/pricing-table/pricing-table.js]: just removed

File: express/blocks/pricing-table/pricing-table.css [hunk 2]
```diff
@@ -435,6 +443,11 @@ main .section .pricing-table-wrapper:has(.pricing-table.many-cols) {
 .pricing-table .row.section-row .col .col-content {
   font-weight: 600;
 }
+
+.pricing-table .row.section-row .col .col-content em{
```
Comment (kennethlum) [inline express/blocks/pricing-table/pricing-table.css:447]: small thing. space after `em`

**Review by kennethlum — APPROVED** [review]

**Review by fullcolorcoder — APPROVED** [review]

**Comment by kennethlum** [comment]
Is the Figma / spec the following? It may be able to use a little left and right padding:
Screenshot 2024-09-18 at 1 03 58 PM

Screenshot below uses left and right 25px padding:
Screenshot 2024-09-18 at 1 02 39 PM

Also the Educators and Nonprofits footnotes font being smaller:
Screenshot 2024-09-18 at 1 03 32 PM

**Review by JingleH — APPROVED** [review]

**Pull request merged**